.projenrc.js

```diff
@@ -194,6 +191,13 @@ new javascript.UpgradeDependencies(project, {
   },
 });
 
+project.tasks
+  .tryFind("docgen")
+  .reset(
+    "jsii-docgen .jsii -o docusaurus/docs/api/projen --split-by-submodule"
+  );
+project.gitignore.addPatterns("docusaurus/docs/api/");
```

Comment:
This one is rather important ;)

```suggestion
project.npmignore.exclude("/docusaurus/");
```